README.rst

```diff
@@ -91,7 +91,7 @@ On your ``settings.py``.
 Where the settings data are stored?
 -----------------------------------
 
-*Decouple* supports both *.ini* and *.env* files.
+*Decouple* supports both *.ini*, *.yaml* and *.env* files.
```

Comment:
The word "both" should probably be removed here, since the list is now more than two file types.